src/main/java/games/strategy/engine/data/CompositeRouteFinder.java

```diff
@@ -15,7 +16,7 @@
   private static final Logger s_logger = Logger.getLogger(CompositeRouteFinder.class.getName());
 
   private final GameMap m_map;
-  private final HashMap<Match<Territory>, Integer> m_matches;
+  private final Map<Match<Territory>, Integer> m_matches;
```

Comment:
@ron-murhammer Good question, and I thought the same thing when I made this change.  However, I did not check compatibility because `CompositeRouteFinder` does not appear to be serializable, nor could I find any reflective uses.

In fact, the only use of this class is in the `GameMap#getCompositeRoute()` method, where it's only used locally.  Basically, it looks like `CompositeRouteFinder` is a _Method Object_ (i.e. a utility) to encapsulate the relatively complex route-finding logic.

Let me know if that doesn't sound right, and I'll run through the compatibility tests.  Or, because I don't think the cost of running all those tests is worth the bang of this one change, I'd prefer to just revert this line. :smile: